### ISSUE TITLE
Audio Focus Settings for Mobile Exports

### ISSUE THREAD

**Issue by BluePinStudio** [Scirra/Construct-feature-requests#392]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

At the moment, with the built-in exporter and default settings both for iOS and Android, you will get a game with a glaring problem. If a user is listening to music or a podcast and they launch our games, their music or podcast will be muted because the app assumes audio focus.

This is problematic because users want to listen to their own music while playing our games. 

### Possible workarounds or alternatives

There are ways around it by fiddling around in xcode and android studio. This involves adding java or c code to the project and I haven't verified if it works.

### Proposed solution

A checkbox on the audio object or in the export to disable audio focusing would be fantastic! That way users can use the Scirra app bundler and default Xcode settings :)

### Why is this idea important?

Users are leaving negative reviews on games made with Construct due to this problem. Their podcast or music gets interrupted by our game and they don't want a game that doesn't allow them to listen to their own audio while playing.

**Comment by Bob-Hate** [comment]
Yes please make the focus optional, so people can listen their own music.
